src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.WindowHelper.cs

```diff
@@ -31,25 +33,12 @@ public static class WindowHelper
 
 			public static bool IsXamlIsland { get; set; }
 
-			public static Windows.UI.Xaml.Window CurrentTestWindow
```

Comment:
I would keep the ability to test the test window - we could potentially want to run the tests in a secondary window, in which case this should be settable